docs/source/getting-started.md

```diff
@@ -29,7 +29,7 @@ The following are required to install the ExecuTorch host libraries, needed to e
 <hr/>
 
 ## Preparing the Model
-Exporting is the process of taking a PyTorch model and converting it to the .pte file format used by the ExecuTorch runtime. This is done using Python APIs. PTE files for common models can be found on HuggingFace (TODO add link).
```

Comment:
Do you want to link to colab notebook for example model preparation? 

https://colab.research.google.com/drive/1qpxrXC3YdJQzly3mRg-4ayYiOjC6rue3?usp=sharing